of-builder/Dockerfile

```diff
@@ -8,18 +8,17 @@ RUN go build -o /usr/bin/of-builder .
 
 FROM alpine:3.7
 
-RUN addgroup -S app && adduser -S -g app app
-RUN mkdir -p /home/app
+RUN addgroup -S app && adduser app -S -G app \
```

Comment:
Adding the user to a group is causing an "operation not permitted" when untar tries to lchown in the tmp folder, e.g. "/tmp/buildctx584615953/context".